app/helpers/application_helper.rb

```diff
@@ -1,17 +1,18 @@
+# frozen_string_literal: true
 module ApplicationHelper
   def flash_class(level)
     case level
-      when :notice then ''
-      when :error then 'flash-error'
-      when :alert then 'flash-warn'
+    when :notice then ''
+    when :error then 'flash-error'
+    when :alert then 'flash-warn'
     end
   end
 
   def sidemenu_link_to(text, path, options = {})
-    css_class = "menu-item"
-    css_class += " selected" if current_page?(path)
+    css_class = 'menu-item'
+    css_class += ' selected' if current_page?(path)
     options.merge!(class: css_class)
-    if icon = options[:icon]
+    if (icon = options[:icon]).present?
```

Comment:
do not assign vars in if
make it
```ruby
if options[:icon].present?
```
and then in if body use options[:icon] instead of icon var